src/qiskit_iqm/fake_backends/iqm_fake_backend.py

```diff
@@ -67,6 +75,7 @@ class IQMErrorProfile:
     two_qubit_gate_depolarizing_error_parameters: dict[str, dict[tuple[str, str], float]]
     single_qubit_gate_durations: dict[str, float]
     two_qubit_gate_durations: dict[str, float]
+    readout_errors: dict[str, dict[str, float]]
```

Comment:
`dict[str, dict[str, float]]` or `dict[str, dict[int, float]]`? I am torn 🤔

Reply:
Imho string is better if there is a interest to add error rates for "00" readout later (to see the qiskit way of doing it, check here: https://qiskit.org/documentation/aer/stubs/qiskit_aer.noise.ReadoutError.html